Formula/mrboom.rb

```diff
@@ -0,0 +1,19 @@
+class Mrboom < Formula
+  desc "Mr.Boom is an 8 player Bomberman clone"
+  homepage "http://mrboom.mumblecore.org"
```

Comment:
http://mrboom.mumblecore.org/

Reply:
Yes ? Any clue whats wrong, it fails on 
install -m 555 mrboom.out /usr/local/Cellar/mrboom/3.8/bin/mrboom

Reply:
>Yes ?

should have trailing slash

>Any clue whats wrong, it fails on
install -m 555 mrboom.out /usr/local/Cellar/mrboom/3.8/bin/mrboom

It's a Makfile bug. It's assuming `BINDIR` already exists, which is not the case.

It needs
```
diff --git a/Makefile b/Makefile
index 80a8e95..d795d19 100644
--- a/Makefile
+++ b/Makefile
@@ -256,7 +256,9 @@ strip:
 	$(STRIP) $(TARGET_NAME).out
 
 install: strip
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(PREFIX)/$(BINDIR)
 	$(INSTALL) -m 555 $(TARGET_NAME).out $(DESTDIR)$(PREFIX)/$(BINDIR)/$(TARGET_NAME)
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(PREFIX)/$(MANDIR)
 	$(INSTALL) -m 644 Assets/$(TARGET_NAME).6 $(DESTDIR)$(PREFIX)/$(MANDIR) 
 
 .PHONY: clean
```
and it's actually `MANDIR=share/man/man6` that you need because it does it with relative paths, which is quite weird, but :man_shrugging:

Reply:
Thanks, do you have an example on how to modify the formula to include that patch ?

Reply:
You'll need to commit upstream and then use a `patch do` block like this:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/tiger-vnc.rb#L21-L27

or just tag a new release